src/App.js

```diff
@@ -0,0 +1,100 @@
+import React, {Component} from 'react';
+import _ from 'lodash';
+import * as apiClient from './http/apiClient';
+import {newList} from './factories';
+
+import ListsView from './components/ListsView';
+import ListView from './components/ListView';
+
+export default class App extends Component {
```

Comment:
I would highly recommend using functional components wherever possible and then ES6 classes when you need state / lifecycle hooks. createClass is a react specific thing; ES6 is pure JS / the future.

The release notes explain why I feel this way:

https://facebook.github.io/react/blog/2015/10/07/react-v0.14.html

"In idiomatic React code, most of the components you write will be stateless, simply composing other components."

"This pattern is designed to encourage the creation of these simple components that should comprise large portions of your apps. In the future, we’ll also be able to make performance optimizations specific to these components by avoiding unnecessary checks and memory allocations."

https://facebook.github.io/react/blog/2015/01/27/react-v0.13.0-beta-1.html

"We figured that we're not in the business of designing a class system. We just want to use whatever is the idiomatic JavaScript way of creating classes.
In React 0.13.0 you no longer need to use React.createClass to create React components."